orchestra/admin.py

```diff
@@ -0,0 +1,190 @@
+from django.contrib import admin
+from django.core.urlresolvers import reverse
+from django.utils.html import format_html
+
+from orchestra.models import Certification
+from orchestra.models import Iteration
+from orchestra.models import PayRate
+from orchestra.models import Project
+from orchestra.models import Step
+from orchestra.models import Task
+from orchestra.models import TaskAssignment
+from orchestra.models import TimeEntry
+from orchestra.models import Worker
+from orchestra.models import WorkerCertification
+from orchestra.models import Workflow
+from orchestra.models import WorkflowVersion
+
+admin.site.site_header = 'Orchestra'
+admin.site.site_title = 'Orchestra'
+admin.site.index_title = 'Orchestra'
+
+
```

Comment:
sanity check---the number of admin.register reigstrations is the same as it was before this PR?